### ISSUE TITLE
xmpp: Better display when domain name not set

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#550]
When the domain name is not set, the XMPP page shows this:
```
Your XMPP server domain is set to . User IDs will look like username@.
```

Instead we should just say something like "Your XMPP server domain is not set".

**Comment by Xavier-J-Ortiz** [comment]
Hello all!

For the record, this would be my very very first bug to be squashed on an opensource project... ever! I feel confident about doing this, but the unknown always seems to have a "Here be dragons sign".

I'd like to take a stab at this at this if the terms are alright. :)

**Comment by mridulnagpal** [comment]
I am a student of IIIT-H and I would like to work on this issue

**Comment by jvalleroy** [comment]
Fixed in #633.
